Exercise_E.js

```diff
@@ -35,6 +35,16 @@ var KEYS = {
 
 function maya(number) {
   // CODE HERE
+  let splitNumber = number.toString().split("") //"423" => ["4", "2", "3"];
+  let string = ""
+  
+  for(let i = 0; i <= string.length; i++) {
+    string += KEYS[splitNumber[i]]
+
+  } return string;
+  
 }
 
+
 console.log(maya(423));
+

```

Comment:
For this exercise it will not run, as first you have no added anything inside the string. But as you are converting the data to a string and then splitting it. You could need to use `for(let i = 0; i < splitNumber.length; i++) {
    string += KEYS[splitNumber[i]]

  } return string;` and then it would run properly.